cubi_isa_templates/isatab-sc_rnaseq-multiplex/cookiecutter.json

```diff
@@ -0,0 +1,127 @@
+{
+  "investigation_title": "Investigation_Title",
+  "sample_names": "alpha,beta,gamma",
+  "a_measurement_type": "transcription profiling",
+  "_measurement_types": {
+      "transcription profiling" : {
+         "accession" : "http://purl.obolibrary.org/obo/OBI_0000424",
+         "source" : "OBI"
+      }
+  },
+  "a_technology_type": "nucleotide_sequencing",
+  "_technology_types": {
+      "nucleotide_sequencing" : {
+         "accession" : "http://purl.obolibrary.org/obo/OBI_0000626",
+         "source" : "OBI"
+      }
+  },
```

Comment:
These should be fixed as this is a sc-rnaseq template. Use private variables to not prompt the user for input or just set these values as strings in the ISA template files.

https://cookiecutter.readthedocs.io/en/stable/advanced/private_variables.html